mapstory/templates/account/email/welcome_message.html

```diff
@@ -2,80 +2,126 @@
+                                <h2 style="color:#f46817; font-size:24px; font-weight:bold;">Welcome to the MapStory community!</h2>
+
+                                <p style="margin:8px 0">
+                                    Hi, {% firstof confirmation.email_address.user.first_name confirmation.email_address.user.username %}!</p>
+
+                                <p style="margin:8px 0">By creating an account, you've joined a community of global citizens working together to
+                                    organize data and tell stories about how and why our world has changed, is changing, and might change over time.
+                                    We come from different places and have different interests, but we are connected by our belief that knowledge is
+                                    meant to be shared and can always be improved!</p>
+
+                                <iframe width="560" height="315" src="https://www.youtube.com/embed/Jis7zXhC4gU" frameborder="0"
+                                        allowfullscreen></iframe>
+                                <p style="margin:8px 0">Here's how you can get started:</p>
+
+                                <h3 style="color:#f46817; font-size:18px; font-weight:normal; margin:32px 0 8px 0;">Learn</h3>
+
+                                <p style="margin:8px 0">MapStorytelling involves using open data, maps and your own reflection to illuminate change in
+                                    place over time. It's something anyone can learn, and your peers are adding more resources all the time to help
+                                    you along.</p>
+
+                                <p style="margin:8px 0"><a href="http://www.mapstory.org/{% url 'getpage' 'skills' %}">Learn the <span
+                                        style="color:#f46817 !important;">Basics</span></a>.</p>
+
+                                <h3 style="color:#f46817; font-size:18px; font-weight:normal; margin:32px 0 8px 0;">Share What You Know</h3>
+
+                                <p style="margin:8px 0">There's three ways to share on MapStory. Just remember <b>A, B, C</b>. First, you can
+                                    <b>add</b> new spatially and temporally enabled datasets, which we call StoryLayers. Second, you can <b>build</b>
+                                    MapStories using StoryLayers and other narrative elements that you would like. Finally, you can <b>curate</b>
+                                    content that appears on MapStory to improve its quality and accuracy.</p>
+
+                                <p style="margin:8px 0">Start <a style="color:#f46817" href="http://mapstory.org/">sharing on MapStory</a>.</p>
+
+                                <h3 style="color:#f46817; font-size:18px; font-weight:normal; margin:32px 0 8px 0;"><a
+                                        href="http://www.mapstory.org/{% url 'getpage' 'involved' %}">Build the Community</a></h3>
```

Comment:
See pull request for the best way to handle site urls https://github.com/MapStory/mapstory-geonode/pull/194 {{ current_site.url }}